Libraries/LibGfx/Point.h

```diff
@@ -208,6 +208,11 @@ class Point {
         return sqrtf(powf(m_x - other.m_x, 2.0f) + powf(m_y - other.m_y, 2.0f));
     }
 
+    Point absolute_relative_distance_to(const Point& other) const
```

Comment:
This name is weird, given that "absolute" and "relative" are antonyms.

Reply:
...I didn't even think how odd that sounds
My name-fu is not that strong, it seems. Oops.